shared/ios/Keybase/ItemProviderHelper.m

```diff
@@ -311,13 +311,23 @@ - (void)processItem:(NSItemProvider*)item {
   
   if ([item hasItemConformingToTypeIdentifier:@"public.image"]) {
     if (self.isShare) {
+      [decodes addObject:^(){
+        [item loadItemForTypeIdentifier:@"public.image" options:nil completionHandler:imageHandler];
```

Comment:
ios will try and coorce the data dependent on the type of the parameters in the blocks, so in this flow (sometimes) you get an image and not a url, so we have to hit the image handler instead else it'll error